uvicorn/protocols/http/h11_impl.py

```diff
@@ -477,6 +482,27 @@ async def send(self, message):
                 output = self.conn.send(event)
                 self.transport.write(output)
 
+                self.timing.response_ended()
+
+                if self.access_log:
+                    if self.access_log_format is None:
+                        self.access_logger.info(
+                            '%s - "%s %s HTTP/%s" %d',
+                            get_client_addr(self.scope),
+                            self.scope["method"],
+                            get_path_with_query_string(self.scope),
+                            self.scope["http_version"],
+                            self.access_log_fields.status_code,
+                        )
+                    else:
+                        try:
+                            self.access_logger.info(
+                                self.access_log_format,
+                                self.access_log_fields,
+                            )
+                        except:  # noqa
+                            self.logger.error(traceback.format_exc())
```

Comment:
When can this happen? Only when the format specified is not valid?

I guess we need to check what/how `gunicorn` and `hypercorn` behaves in cases like this? :thinking:

Reply:
This is exactly what gunicorn does:

https://github.com/benoitc/gunicorn/blob/cf55d2cec277f220ebd605989ce78ad1bb553c46/gunicorn/glogging.py#L348-L351

Reply:
Re: when can it happen: it means a failure in log formatter or handler to trigger this, e.g. when access log is written to a file that consumes all space in the storage, or when the custom formatter specified in logging config for `uvicorn.access` logger is buggy.

Reply:
It's fine then. Thanks!